hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsBackoffMetrics.java

```diff
@@ -305,7 +328,9 @@ public String toString() {
       return EMPTY_STRING;
     }
     StringBuilder metricBuilder = new StringBuilder();
-    getRetryMetrics(metricBuilder);
+    if (isRetryMetricEnabled) {
```

Comment:
I think we should always have retry metrics added as a part of these aggregate metrics